app/templates/teachers/teacher-student-view.jade

```diff
@@ -364,15 +365,22 @@ mixin breadcrumbs
                               .small!= marked(solution.description || '')
                           tr
                             td
-                              if view.levelStudentCodeMap[level.get('original')]
+                              - var codeMaps = view.levelStudentCodeMap[level.get('original')].filter(m => m)
+                              - var multipleCodes = false
+                              case codeMaps.length
```

Comment:
And then we can avoid the `case` here and just do `if`/`else` on codeMaps.length on whether we show `teacher.no_code_yet`.